integration/sync/test_collision-handlers.js

```diff
@@ -0,0 +1,115 @@
+var MongoClient = require('mongodb').MongoClient;
+var assert = require('assert');
+var sinon = require('sinon');
+var _ = require('underscore');
+var defaultDataHandlersModule = require('../../lib/sync/default-dataHandlers');
+var dataHandlersModule = require('../../lib/sync/dataHandlers');
+
+var DATASETID = "collisionHandlersTest";
+var MONGODB_URL = "mongodb://127.0.0.1:27017/test";
+
+var mongodb;
+var dataHandlers;
+var defaultHandlers;
+
+function getCollisionData() {
+  return {
+    pre: { test: 'test1' },
+    post: { test: 'test2' },
+    hash: 'testhash',
+    ts: Date.now(),
+    uid: 'testUid'
+  };
+}
+
+module.exports = {
+  'before': function(done) {
+    MongoClient.connect(MONGODB_URL, function(err, db){
+      if (err) {
+        console.log('mongodb connection error', err);
+        return done(err);
+      }
+      mongodb = db;
+      mongodb.dropCollection(DATASETID + '_collision');
+      defaultHandlers = defaultDataHandlersModule(mongodb);
+      dataHandlers = dataHandlersModule({
+        defaultHandlers: defaultHandlers
+      });
+      return done(err);
+    });
+  },
+  'afterEach': function(done) {
```

Comment:
similarly to the above, put this inside another object. Otherwise this will run before every test